scala/spark-wordcount/pom.xml

```diff
@@ -9,11 +9,11 @@
     <version>0.1</version>
 
     <properties>
-        <bigtable.version>1.7.0</bigtable.version>
-        <hbase.version>1.3.1</hbase.version>
+        <bigtable.version>1.15.0</bigtable.version>
+        <hbase.version>1.6.0</hbase.version>
```

Comment:
why 1.6?

Reply:
That's the latest in 1.x line to my understanding. I didn't want to upgrade this too "high" which may have led to some other changes.

We could have a separate profile for HBase 2.x with all the needed dependency changes. I'd do it in a separate PR though. WDYT?

Reply:
java-bigtable-hbase targets 1.4.12, I think the shc adapter targets 2.0.4, why introduce a 3rd version?

Reply:
Just to show that the combo works fine? I'm fine to downgrade if requested. Please note that the sample app (`spark-wordcount`) does not use the shc adapter...yet.

What `hbase.version` to use for the sample? Please guide @igorbernstein2 🙏

Reply:
Looking at dataproc, it seems like it targets hbase 1.5.0. We should upgrade java-bigtable-hbase to target that as well and use that version here

Reply:
done